docs/design/coreclr/botr/porting-ryujit.md

```diff
@@ -51,29 +112,32 @@ _TARGET_AMD64_, _TARGET_X86_, _TARGET_ARM64_, _TARGET_ARM_
 * Target.h
 * InstrsXXX.h
 
-## Instruction Encoding
-* The instrDesc is the data structure used for encoding
+# Instruction Encoding
+
+* The `insGroup` and `instrDesc` the data structures used for encoding
```

Comment:
should be "are the data structures" ?
And the line below should probably be "The `instrDesc` is initialized .."